src/WebJobs.Script.WebHost/Routing/ScriptRouteHandler.cs

```diff
@@ -53,6 +53,12 @@ public Task InvokeAsync(HttpContext context, string functionName)
                 }
             }
 
+            if (context.Request.Path.StartsWithSegments(new PathString("/admin"), System.StringComparison.OrdinalIgnoreCase))
```

Comment:
So the regression was in https://github.com/Azure/azure-webjobs-sdk-extensions/pull/668 where this issue was actually discussed? The /admin guard was removed in that PR.

Reply:
And previously we had added that code to address this very issue (fix for https://github.com/Azure/azure-functions-host/issues/2590)

Reply:
yes I actually needed to add that back and exclude admin/warmup. see other PR I just sent: https://github.com/Azure/azure-webjobs-sdk-extensions/pull/696

if we don't do it there then the status code will be 200 instead of 401 even though no http trigger function gets executed.